install_files/ansible-base/ansible.cfg

```diff
@@ -2,12 +2,12 @@
 #ask_pass=False
 #ask_sudo_pass=False
 display_skipped_hosts=False
-timeout=60
+timeout=120
 # Use dynamic inventory script to determine local IPv4 addresses
 # from site-specific vars, or Onion URLs for SSH over ATHS.
 inventory=inventory-dynamic
 
 [ssh_connection]
 scp_if_ssh=True
-ssh_args = -o ControlMaster=auto -o ControlPersist=600s -o ConnectTimeout=60
+ssh_args = -o ControlMaster=auto -o ControlPersist=1200
```

Comment:
This still seems backward to me -- I thought we'd seen problems due to lingering connections after reboot that required manual cleanup, which I'd think a shorter ControlPersist value might help -- but I can't argue with the results.